libs/jwst-rpc/src/broadcast.rs

```diff
@@ -50,15 +50,19 @@ pub async fn subscribe(workspace: &mut Workspace, identifier: String, sender: Br
                             .send(BroadcastType::BroadcastAwareness(update.clone()))
                             .is_err()
                         {
-                            info!("broadcast channel {workspace_id} has been closed",)
+                            debug!("broadcast channel {workspace_id} has been closed",)
                         }
                         dedup_cache.insert(update, ());
                     }
                 }
             })
-            .await
+            .await;
+
+        // TODO: this is a hack to prevent the subscription from being dropped
+        // just keep the ownership
+        std::mem::forget(awareness);
```

Comment:
memory leak?

Reply:
missed in the refactoring, should be store in the Workspace instance to keep the drop timing